src/Utilities/_DataValidation.php

```diff
@@ -198,9 +198,7 @@ function validateSortedSetRanks(?int $startRank, ?int $endRank): void
     function validateSortedSetElements(array $elements): void
     {
         foreach ($elements as $value => $score) {
-            if (!is_float($score)) {
-                throw new InvalidArgumentError("Sorted set score must be a float");
-            }
+            validateSortedSetScore($score);
```

Comment:
👍

Reply:
fetchByScore and lengthByScore already call validateSortedSetScores, which I modified to validate the types.